tests/database/test_provider_db_accessor.py

```diff
@@ -26,54 +26,87 @@
 class ProviderDBAccessorTest(MasuTestCase):
     """Test Cases for the ProviderDBAccessor object."""
 
-    def test_initializer(self):
-        """Test Initializer"""
-        uuid = '6e212746-484a-40cd-bba0-09a19d132d64'
+    def test_initializer_provider_uuid(self):
+        """Test Initializer with provider uuid."""
+        uuid = self.aws_test_provider_uuid
         accessor = ProviderDBAccessor(uuid)
         self.assertIsNotNone(accessor._session)
         self.assertTrue(accessor.does_db_entry_exist())
         accessor.close_session()
 
+    def test_initializer_auth_id(self):
+        """Test Initializer with authentication database id."""
+        auth_id = self.aws_db_auth_id
+        accessor = ProviderDBAccessor(auth_id=auth_id)
+        self.assertIsNotNone(accessor._session)
+        self.assertTrue(accessor.does_db_entry_exist())
+        accessor.close_session()
```

Comment:
Is it necessary to call close_session() at the end of each test?  Shouldn't the `tearDown()` method handle that?

Reply:
Yeah we have a database context manager now that could also be used.  There's several examples in the test code that could be moved to it.  I just didn't bother at this point since it works and my change was big enough...